### PR TITLE
Add some utf8 branch

### PR DESCRIPTION
**Pull request by siyuan0322**

### CODE REVIEW

File: modules/basic/ds/arrow_utils.cc [hunk 1]
```diff
@@ -77,7 +77,7 @@ std::shared_ptr<Column> CreateColumn(
     return std::make_shared<DoubleColumn>(chunked_array, chunk_size);
   } else if (type == arrow::large_utf8()) {
     return std::make_shared<StringColumn>(chunked_array, chunk_size);
-  } else if (type == arrow::binary()) {
+  } else if (type == arrow::large_binary()) {
```
Comment (sighingnow) [inline modules/basic/ds/arrow_utils.cc:80]: Could we keep the both case here?

**Review by sighingnow — APPROVED** [review]
Looks good to me, but I'm wondering if this PR is ready for merging?

File: modules/basic/ds/arrow_utils.cc (hunk 1 above)
Comment (siyuan0322) [reply modules/basic/ds/arrow_utils.cc:80]: StringColumn's ArrayType is declared like `ConvertToArrowType<std::string>`.

Comment (sighingnow) [reply modules/basic/ds/arrow_utils.cc:80]: I see. Thanks for the clearification.

**Pull request merged**